### ISSUE TITLE
Anyone having issues with when approving requests since update

### ISSUE THREAD

**Issue by Baltimorepc** [Ombi-app/Ombi#5202]
### Summary

When someone requests a movie and i approve, now it is approved but never shows on radarr. I get messages on my ombi app and emails saying the movie was requested but could not be added. This has been added to the request queue and will keep trying.  Never adds. this worked fine before i updated

### Ombi Version

 4.44.1

### What platform(s) does this occur on?

Docker

### What database are you using?

SQLite (Default)

**Comment by TheRC51** [comment]
Yeah i'm having the same now. Using the LinuxServer Docker container from the dev build. Version reported is 4.46.7. Seeing odd logs about it not being able to authenticate to Radarr but when I run the tests it works

**Comment by emogu2** [comment]
Getting a similar issue here too, except when I run the authentication tests it says could not connect to Radarr. Sonarr works fine and Radarr did too until this afternoon when I started getting emails saying movies were requested but couldn't be added. No connectivity issues in 2 years, then it just stopped this afternoon. Radarr is run as an app on a Mac and Ombi is in a docker container from linuxserver. Everything updated to latest official builds (Ombi at 4.44.1). I've tried starting/stopping Radarr and deleting and pulling a fresh Ombi container, with no change.

Logs show: [ERR] Failed caching queued items from Radarr
System.Net.Http.HttpRequestException: An error occurred while sending the request.

**Comment by emogu2** [comment]
I solved my issue, so see if this works for you. Turns out it was a recent Radarr update that messed things up. For running Radarr on macOS, you need to go into the terminal and run a quick command to self-sign the app:
`codesign --force --deep -s - /Applications/Radarr.app && xattr -rd com.apple.quarantine /Applications/Radarr.app`
Radarr usually updates fine on its own without needing to manually complete this step each time, but this time it needed it for me. So I downloaded a fresh version of the app, did the code signing, and tested the Ombi connection and it was successful. So if you happen to be running Radarr on MacOS, give that a shot.

**Comment by Baltimorepc** [comment]
im using the liuxserver docker on linux/docker and still have the issue.

**Comment by TheRC51** [comment]
I forgot to update, using an fqdn didn’t work, but if you put the IP address of your radarr instance it worked for me. Try that

**Comment by Baltimorepc** [comment]
thats how i have it setup with ip says its connected but ombiu still sending me messages  adding to retry queue and it never gets added
